server/webhook/jobs.go

```diff
@@ -0,0 +1,66 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/xanzy/go-gitlab"
+)
+
+func (w *webhook) HandleJobs(ctx context.Context, event *gitlab.JobEvent) ([]*HandleWebhook, error) {
+	handlers, err := w.handleChannelJob(ctx, event)
+	if err != nil {
+		return nil, err
+	}
+	return cleanWebhookHandlers(handlers), nil
+}
+
+func (w *webhook) handleChannelJob(ctx context.Context, event *gitlab.JobEvent) ([]*HandleWebhook, error) {
+	senderGitlabUsername := event.User.Name
+	repo := event.Repository
+	res := []*HandleWebhook{}
+	message := ""
+	message = fmt.Sprintf("### Pipeline Job Stage: **%s**\n", event.BuildStage)
```

Comment:
Should we also display info about this repo in this message, or is it already present? Could you post some screenshots of the output here?

Reply:
You can have a look at the video in the header of Gitlab channel.

Reply:
I think we should mention the repo name in the output. Current screenshot from the video you mentioned:

<img width="460" alt="image" src="https://user-images.githubusercontent.com/6913320/190947632-3acc6c12-b3e2-4199-a60f-43a4dfd3b747.png">

Reply:
sure that's easy.